examples/bun-rl/index.ts

```diff
@@ -16,30 +20,44 @@ const aj = arcjet({
 export default {
   port: 3000,
   fetch: aj.handler(async (req) => {
-    const decision = await aj.protect(req);
+    const userId = "user123"; // Replace with your authenticated user ID
+    const decision = await aj.protect(req, { userId, requested: 5 }); // Deduct 5 tokens from the bucket
     console.log("Arcjet request ID", decision.id);
     console.log("Arcjet decision", decision.conclusion);
 
     if (decision.isDenied()) {
-      return new Response("Blocked", { status: 403 });
+      if (decision.reason.isRateLimit()) {
+        return new Response("Too many requests", { status: 429 });
+      } else {
+        return new Response("Forbidden", { status: 403 });
+      }
     }
 
     return new Response("Hello world");
   }),
 };
 
 // Or using the `Bun.serve()` API
-// const server = Bun.serve({
-//   port: 3000,
-//   fetch: aj.handler(async (req) => {
-//     const decision = await aj.protect(req);
+/*
```

Comment:
The keyboard shortcut for "uncomment all lines" in VSCode works nicer with line comments than block comments.

Reply:
You can use toggle block comment shortcut as well. Removing this style of comment is easier if you don't know the shortcut.